dashboard/app/views/levels/editors/_weblab.html.haml

```diff
@@ -4,6 +4,14 @@
     Sources
 = f.text_area :start_sources, placeholder: 'Starting sources', rows: 40, class: "input-block-level", value: @level.start_sources
 .field
+  = f.label :submittable, 'Submittable'
+  %p
+    If checked, students with teachers can "submit" a solution for
+    grading. Submitting a solution means that it is marked as
+    submitted and they can no longer edit it (unless a teacher returns
```

Comment:
yeah, now they can. I just copy/pasted from applab. that's probably the source of my confusion 😄